spec/run_oc_pedant.rb

```diff
@@ -83,10 +83,7 @@ def start_cheffs_server(chef_repo_path)
       '--skip-users',
       '--skip-organizations',
       '--skip-multiuser',
-
-      # will be supported.
-      '--skip-policies',
-      '--skip-cookbook-artifacts',
+      '--skip-chef-zero-quirks',
```

Comment:
In keeping with Dan's excellent doc comments about the various skips and why we're doing them, can you add a description for `chef-zero-quirks`?